lib/Humming-Bird/Core.rakumod

```diff
@@ -255,13 +191,19 @@ class Response is HTTPAction is export {
 
     # Set a file to output.
     method file(Str:D $file --> Response) {
-        $.write($file.IO.slurp || '', 'text/plain'); # TODO: Infer type of output based on file extension
+        $.write($file.IO.slurp, $mime.type($file.split('.', :skip-empty)[*-1] // 'text/plain')); # TODO: Infer type of output based on file extension
```

Comment:
If I understand well, the middle of the line tests for extension. Is there a reason you aren't using [IO::Path's extension method](https://docs.raku.org/routine/extension#(IO::Path)_method_extension)? (It can work on non-existent paths, of course! :P)

Reply:
Did not know that was a thing. Thanks!